lib/pod/command/why.rb

```diff
@@ -195,7 +196,7 @@ def find_dependencies(source, graph, reverse, direct)
         end
 
         if sorted_dependencies.empty?
-          UI.puts 'Nothing'
+          UI.puts 'No dependencies found'
```

Comment:
Changed this because if stderr is sent to `/dev/null`, users won't see the `What.+depend.+?` question, and `Nothing` won't make sense. Since we are already making a breaking change, it's fine to change this in the process.